openscenario/openscenario_interpreter/src/syntax/act.cpp

```diff
@@ -31,6 +32,19 @@ Act::Act(const pugi::xml_node & node, Scope & scope)
   });
 }
 
+auto Act::run() -> void
+{
+  for (size_t index{0}; auto && maneuver_group : elements) {
```

Comment:
This repository is compiled with C++17. Don't take advantage of C++20 features yet.

```suggestion
  std::size_t index = 0;
  for (auto && maneuver_group : elements) {
```